tenderly/.env

```diff
@@ -0,0 +1,6 @@
+PROXY_ETHEREUM_NODE_PORT=8080
+PROXY_ETHEREUM_NODE_URL=https://daemon.ethereum.shapeshift.com
+PROXY_UNCHAINED_ETHEREUM_HTTP_PORT=8081
+PROXY_UNCHAINED_ETHEREUM_HTTP_URL=https://dev-api.ethereum.shapeshift.com
+PROXY_UNCHAINED_ETHEREUM_WS_PORT=8082
+PROXY_UNCHAINED_ETHEREUM_WS_URL=wss://dev-api.ethereum.shapeshift.com
```

Comment:
should instead pull these in from main app .env via `dotenv.parse()` to negate a point of config divergence